agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -339,6 +358,32 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
     }
 
     switch (beerocks_header->action_op()) {
+    case beerocks_message::ACTION_APMANAGER_CONFIGURE: {
+        if (m_ap_manager_configured) {
+            break;
+        }
+
+        LOG(TRACE) << "ACTION_APMANAGER_CONFIGURE";
+
+        auto config = beerocks_header->addClass<beerocks_message::cACTION_APMANAGER_CONFIGURE>();
+        if (!config) {
+            LOG(ERROR) << "addClass cACTION_APMANAGER_CONFIGURE failed";
+            return false;
+        }
+
+        low_filter   = config->iface_filter_low();
```

Comment:
This doesn't seem to be used anywhere, maybe remove it?

Reply:
https://github.com/prplfoundation/prplMesh/pull/1271/commits/df685422eca9a264d286a59d5c417db9f1032a93#r426739414